config/tech-docs.yml

```diff
@@ -5,7 +5,7 @@ host: https://content-api.publishing.service.gov.uk
 show_govuk_logo: true
 service_name: Content API
 service_link: /
-phase: Work In Progress
+phase: Beta
```

Comment:
If possible it'd be good to separate this change from the feedback info. As we probably want to keep the switch to Beta as a distinct PR ready to merge when we decide "yup it's not a WIP" - which is hopefully very soon.